rfcs/0194-flake-entrypoint.md

```diff
@@ -0,0 +1,389 @@
+---
+feature: flake-entrypoint
+start-date: 2025-12-08
+author: Robert Hensing (@roberth)
+co-authors: (to be determined)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Reduce boilerplate in flakes by changing how Nix acquires flake outputs,
```

Comment:
So the vision is to use a project like naersk or pyproject2nix with either modifications or a wrapper flake that provides callFlakeEntrypoint and if the stars align you only need the flake.toml?

Makes TOML for RFC 193 more appealing to me.